application/src/main/java/org/thingsboard/server/actors/ruleChain/RuleChainActorMessageProcessor.java

```diff
@@ -396,9 +397,8 @@ private void pushMsgToNode(RuleNodeCtx nodeCtx, TbMsg msg, String fromRelationTy
     }
 
     @Override
-    protected RuleNodeException getInactiveException() {
+    protected RuleNodeException getInactiveException(String data, TbMsgMetaData metaData) {
```

Comment:
Do we really need data and metadata in this case?
If rule node or rule chain is not active, then real root cause happened previously.

Reply:
But probably we can leave it for debug purposes.